datumaro/plugins/celeba_format.py

```diff
@@ -0,0 +1,151 @@
+
+        super().__init__()
+        self._anno_dir = osp.dirname(path)
+
+        self._categories = { AnnotationType.label: LabelCategories() }
+        self._items = list(self._load_items(path).values())
+
+    def _load_items(self, path):
+        items = {}
+
+        image_dir = osp.join(osp.dirname(self._anno_dir), CelebaPath.IMAGES_DIR)
+        if not osp.isdir(image_dir):
+            image_dir = osp.join(osp.dirname(self._anno_dir), CelebaPath.IMAGES_ALIGN_DIR)
+        if osp.isdir(image_dir):
+            images = {
+                osp.splitext(osp.relpath(p, image_dir))[0].replace('\\', '/'): p
+                for p in find_images(image_dir, recursive=True)
+            }
+        elif osp.isdir(osp.join(osp.dirname(self._anno_dir), CelebaPath.IMAGES_DIR)):
+            images = {}
+
+        label_categories = self._categories[AnnotationType.label]
+        with open(path, encoding='utf-8') as f:
+            for line in f:
+                item_id, item_ann = self.split_annotation(line)
+                label_ids = [int(id) for id in item_ann]
+                anno = []
+                for label in label_ids:
+                    if label > len(label_categories):
+                        for i in range(len(label_categories), label + 1):
+                            label_categories.add('label_%d' % i)
+                    anno.append(Label(label))
+
+                items[item_id] = DatasetItem(id=item_id, image=images.get(item_id),
+                    annotations=anno)
+
+        bbox_path = osp.join(self._anno_dir, CelebaPath.BBOXES_FILE)
+        if osp.isfile(bbox_path):
+            with open(bbox_path, encoding='utf-8') as f:
+                for line in f:
+                    if len(line.split()) < 5 or 'image_id' == line[0:8]:
+                        continue
+                    item_id, item_ann = self.split_annotation(line)
+                    bbox = [float(id) for id in item_ann]
+                    if item_id not in items:
+                        items[item_id] = DatasetItem(id=item_id, image=images.get(item_id))
+                    anno = items[item_id].annotations
+                    label = None
+                    if 0 < len(anno):
+                        label = anno[0].label
+                    anno.append(Bbox(bbox[0], bbox[1], bbox[2], bbox[3], label=label))
+
+        attr_path = osp.join(self._anno_dir, CelebaPath.ATTRS_FILE)
+        if osp.isfile(attr_path):
+            with open(attr_path, encoding='utf-8') as f:
+                f.readline()
+                line = f.readline()
+                name_attr = line.split()
+                attr = {}
+                for line in f:
+                    item_id, item_ann = self.split_annotation(line)
+                    if len(name_attr) != len(item_ann):
+                        continue
+                    for i, ann in enumerate(item_ann):
+                        attr[name_attr[i]] = ann
```

Comment:
```suggestion
                    attr = {name: ann for name, ann in zip(name_attr, item_ann)}
```

Reply:
Also, the attributes are defined as binary, so I think they should be converted to `bool` to make them easier to work with.